docs/tutorials/zero-to-hero.md

```diff
@@ -62,7 +62,7 @@ Now that you're familiar with the various files in your project, let's get start
 
 The first thing that our Oracle Contract must be able to do is read from and write to the blockchain. This way our contract can save external data onto the blockchain for other contracts to interact with.
 
-Navigate to our API [storage docs](/docs/runtime-ts/classes/storage) to review the `setString` and `getString` functions. In later steps we'll show you how to handle more complicated data types.
+Navigate to our API [storage docs](/docs/api/near-sdk-as/classes/_runtime_storage_.storage) to review the `setString` and `getString` functions. In later steps we'll show you how to handle more complicated data types.
```

Comment:
I'm not sure what to link to here, either

Reply:
same as above